functests-render-command/1_render_command/render_test.go

```diff
@@ -97,6 +97,11 @@ func runAndCompare(cmd *exec.Cmd) {
 		Expect(err).ToNot(HaveOccurred())
 
 		res := bytes.Compare(data, refData)
+		if res != 0 {
+			fmt.Fprintf(GinkgoWriter, "files: %q and %q are not identical\n",
+				filepath.Join(refPath, f.Name()),
+				filepath.Join(assetsOutDir, f.Name()))
```

Comment:
better, but I'd actually fix the expect right below.

Reply:
let's keep it like this for now however, until we figure out a smarter diff.

Reply:
the idea in general is that the failing test output should tell us as precisely as possible what failed, instead of forcing us to to go review the expected output. Which is extra-hard because the YAMLs are (kinda-)minified.